console/src/components/NodeForm.tsx

```diff
@@ -25,7 +25,8 @@ const NodeForm = () => {
     clientName,
     source,
     destination,
-    password,
+    name, 
+    client, // todo
```

Comment:
It seemed to me that we don't necessarily need the "client" field to be listed in the fields for the section, so I'm filtering it out here, but something about that felt weird, thus the `// todo`.